front/package.json

```diff
@@ -3,40 +3,41 @@
   "license": "MIT",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve -H 0.0.0.0",
+    "start": "ng serve --host 0.0.0.0",
     "build": "ng build",
-    "build-prod": "ng build --prod --aot=false",
+    "build-prod": "ng build --prod --aot=false --build-optimizer=false",
```

Comment:
perhaps you can remove the option --aot=false now. It was a bug introduce in angular 5 if I remember well. So if you  do you don't need to set the option --build-optimizer to false